### DISCUSSION TITLE
Search Parameter Names

### DISCUSSION THREAD

**Discussion by DrKentB** [gwosc-tutorial/gwosc-catalog#65] [General]
The schema is specifying search parameter names of (far, pastro, snr). However, GWOSC returns to user queries (and uses internally in the database) the following names for search parameters (far, p_astro, network_matched_filter_snr). Only "far" is consistent between the two. At a minimum this should be documented. A discussion of whether or not to change the schema names may be worthwhile also.

Footnote: In order to ingest IAS example events a translation of the inconsistent names was performed. While this may seem efficient, it leaves the user of this schema and the user of GWOSC with an inconsistency that is not ideal. 

FYI: Looking into GRACEDB, it looks as though the values used there are consistent with the schema, i.e., (far, pastro and snr). This might suggest that having the JSON files downloaded from gwosc use the same would be the most consistent solution, but It would likely branch downstream clients that read the JSON and act upon the content based on current nameing convention in gwosc.

**Comment by martinberoiz** [comment]
I moved this one to a discussion until we have a decision and make an issue of it.
